### DISCUSSION TITLE
Is it possible to get the description only via `jj show`

### DISCUSSION THREAD

**Discussion by lazywei** [jj-vcs/jj#2225] [Q&A] [answered]
Hi

Is it possible get the description (i.e., the commit message) only, without other metadata and/or diffs when using `jj show`?
Similar to `git show -s --format=%B`

The reason is I'm trying to create some bash script for me to automate the GitHub branch push & PR creation for the stacked PRs, similar to what Sapling does. And one step requires me to update the PR description with the commit message. So I wonder if there is some easy way to do this programatically.

Right now I'm able to get it by this

```
jj show -r $change_id --git | awk '/Committer:/,/diff --git/' | sed '1d;$d' | sed 's/^[ \t]*//'
```

But it seems it's a bit hacky and not sure if it's reliable enough.

Thanks.

**Comment by martinvonz** [comment, marked as answer]
`jj log --no-graph -T description` should do it.

**Reply by lazywei** [reply]
wow awesome, thank you!

**Reply by martinvonz** [reply]
Sorry about the short answer before (I was on mobile). You can find more information about on https://martinvonz.github.io/jj/latest/templates.html

**Reply by lazywei** [reply]
no worry the `--help` has already done that for me! thanks for pointing out the direction, I was checking the `jj show`'s help doc but couldn't find any templating and didn't realize I looked at a wrong place lol

**Reply by csdogww** [reply]
interesting as you are! haha!

**Reply by ilyagr** [reply]
Perhaps `jj show` should become either a full alias to `jj log` with some template or close to it.

**Comment by GeoffKarnov** [comment]
For those trying to answer this question more recently: 

`jj show --no-patch -T description`

Also now works.

You can add this to your configuration:
```
[aliases]
description = ["show", "--no-patch", "-T", "description"]
```
